conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, tools, CMake
+
+
+class OpenvinoConan(ConanFile):
```

Comment:
I think the original is to contribute these recipes to [Conan center](https://conan.io/center) instead of having them locally. 
how users are expected to use it? they have to clone the repo to use it, but if they cloned - why not to use cmake then?

Is it possible for OpenVINO to upload binary packages https://docs.conan.io/en/latest/introduction.html#binary-management ? Like in all other package manager, who provides binary artifacts.
So, users don't have to build OpenVINO from scratch every time.

In general, what I expect, is:
```sh
conan install openvino/2022.3.0@
```
And it will download prebuilt OpenVINO. If prebuilt OpenVINO is not available for current platform, it should be build from sources

Reply:
As we disscused offline, if we would like to have OV in the Conan center, we need to use 3rd party libraries as a conan packages. After that we can add Conan recipe to the [conan center repository](https://github.com/conan-io/conan-center-index). Finally the OV will be available in [Conan center](https://conan.io/center/)

I removed from the PR sample and recipe but in my opinion the changes in the cmake should be merge as a base to productization Conan, it doesn't affect on the current build systems